aptos-move/framework/aptos-framework/sources/aggregator/aggregator_factory.move

```diff
@@ -73,6 +76,11 @@ module aptos_framework::aggregator_factory {
 
```

Comment:
Can you update initialize_aggregator_factory to use system_addresses:assert_aptos_framework instead of comparing addresses manually?

Reply:
Also, assert!(
            !exists<AggregatorFactory>(signer::address_of(account)),
            error::already_exists(EAGGREGATOR_FACTORY_EXISTS)
        ); is not needed. This would fail in move_to if it already exists